### PR TITLE
Better xenoarch laser gun sprite

### PR DESCRIPTION
**Pull request by Cameron653**
Gives xenoarch guns better sprites.
Also fixes races not getting their pay

### CODE REVIEW

**Comment by Cameron653** [comment]
Adding something

File: code/modules/economy/economy_misc_vr.dm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/hook/startup/proc/pay_raise()
+	economic_species_modifier = list(
```
Comment (Yoshax) [inline code/modules/economy/economy_misc_vr.dm]: If you're making them all equal just remove it entirely or make it a not list and change the line in job.dm to not use the fancy species stuff.

Comment (SpadesNeil) [inline code/modules/economy/economy_misc_vr.dm]: Yea, why not just remove this functionality entirely? There's not much need for it.

File: code/modules/research/xenoarchaeology/finds/finds_eguns_vr.dm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/obj/item/weapon/gun/energy/laser/xenoarch
+	icon = 'icons/obj/xenoarchaeology_vr.dmi'
+
+/obj/item/weapon/gun/energy/laser/practice/xenoarch
+	icon = 'icons/obj/xenoarchaeology_vr.dmi'
+
+/obj/item/weapon/gun/energy/laser/practice/xenoarch //I have zero idea why this is wrote twice. It's how the origonal finds_eguns_vr file was written, so I'm just going to follow its example.
```
Comment (SpadesNeil) [inline code/modules/research/xenoarchaeology/finds/finds_eguns_vr.dm]: This must be a mistake. There's no logical reason why this is defined twice.

**Comment by Cameron653** [comment]
Should be merge-able now.

**Pull request merged**